recipes/skype/webview.js

```diff
@@ -50,7 +50,10 @@ module.exports = (Ferdium, settings) => {
         event.preventDefault();
         event.stopPropagation();
 
-        if (settings.trapLinkClicks === true) {
+        if (url.includes('api.asm.skype.com')) {
+          // Always open file downloads in Ferdium, rather than the external browser
+          window.location.href = url;
```

Comment:
Would it be possible to move this if to its own logic before the trap links?

```
if(url.includes ...) {
...
return
}

... the previous code we had to handle trap links
```

I think this way is cleaner 😁

Reply:
I'm not familiar with the ferdium-app and thought using `if` and `else if` is probably safer, I definitely agree that this is cleaner.